.github/workflows/main.yml

```diff
@@ -49,7 +49,7 @@ jobs:
 
       - uses: actions/setup-python@v3
         with:
-          python-version: 3.10.2
+          python-version: 3.10
```

Comment:
`setup-python` doesn't have Python 3.10.2 for `ubuntu >= 22.04`. (See: https://github.com/pyodide/pytest-pyodide/pull/64). I think it is okay to use python 3.10.X for linting, testing docs, and running host tests of pyodide-build.